examples/sanity-studio/.env.template

```diff
@@ -0,0 +1,3 @@
+# Run `vercel env pull` to generate a .env file from yur Vercel project
```

Comment:
```suggestion
# Run `vercel env pull` to generate a .env file from your Vercel project
```

Reply:
Does `sanity start` work with both `.env` and `.env.local`?

If it works with `.env.local` then we should suggest `vercel env pull .env.local` instead